src/app/authentication/forgot-password.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="col-sm-6 col-sm-offset-3">
+
+    <h1>Reset</h1>
+
+    <div class="alert alert-danger" *ngIf="errorMessage">{{errorMessage}}</div>
+    <div class="alert alert-success" *ngIf="successMessage">{{successMessage}}</div>
+
+    <form [formGroup]="resetForm" (ngSubmit)="onSubmit(resetForm.value)">
+        <div class="form-group input-group" [ngClass]="{'has-error':!resetForm.controls['email'].valid && resetForm.controls['email'].dirty}">
+            <span class="input-group-addon"><i class="glyphicon glyphicon-user"></i></span>              
+            <input type="text" class="form-control" name="email" placeholder="Email" [formControl]="resetForm.controls['email']">
+        </div>      
+        <div class="form-control-feedback">
+            <p *ngIf="resetForm.controls['email'].hasError('required') && (resetForm.controls['email'].dirty)">Email is required</p>
```

Comment:
i meant something like:
```
<p *ngIf="resetForm.controls['email'].hasError() && (resetForm.controls['email'].dirty)">{{ resetForm.controls['email'].errorMessage }}</p>
```
so you don't have to modify the DOM if we ever introduce a 3rd type of error in the future. just a suggestion, totally not necessary when we only have 2 cases